core/net/ipv6/uip-nd6.c

```diff
@@ -517,10 +517,16 @@ na_input(void)
       if(nd6_opt_llao == NULL || !extract_lladdr_from_llao_aligned(&lladdr_aligned)) {
         goto discard;
       }
+
+#if NETSTACK_RADIO != ble_mode
```

Comment:
This change should not be in a core file like this. If the neighbor table is not needed, it would be better to add a generic API for turning off the neighbor table functionality, which would be called from one of the BLE driver files.

Also, `NETSTACK_RADIO` probably is defined to be `ble_mode_driver`, and not `ble_mode`, so this particular `#if` would seem to never trigger anyway. So maybe it isn't needed?